types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -761,7 +761,7 @@ declare namespace Aws {
 declare class Aws {
     constructor(serverless: Serverless, options: Serverless.Options);
 
-    naming: { [key: string]: () => string };
+    naming: { [key: string]: (param: any) => string };
```

Comment:
Thanks for your contribution! I think this could be more precise with `(param?: string)`, WDYT?

Reply:
I'm not sure about all naming function parameters are string or not. But it's ok to set type as `string`.